src/FacadeAccountCreation/FacadeAccountCreation.UnitTests/Core/Services/OrganisationServiceTests.cs

```diff
@@ -432,4 +436,54 @@ public async Task AddSubsidiaryAsync_ShouldReturnOrganisationResponse()
                        req.RequestUri.ToString() == expectedUrl),
             ItExpr.IsAny<CancellationToken>());
     }
+
+    [TestMethod]
+    public async Task GetOrganisationRelationshipsByOrganisationId_Returns_OrganisationRelationshipModel_OnSuccess()
+    {
+        // Arrange
+        var organisationId = Guid.NewGuid();
+
+        var expectedModel = _fixture.Create<OrganisationRelationshipModel>();
+
+        var expectedUrl =
+            $"{BaseAddress}/{OrganisationGetRelationshipUri}/{organisationId}/organisationRelationships";
+
+        _httpMessageHandlerMock.Protected()
+            .Setup<Task<HttpResponseMessage>>("SendAsync",
+                ItExpr.Is<HttpRequestMessage>(x => x.RequestUri != null && x.RequestUri.ToString() == expectedUrl),
+                ItExpr.IsAny<CancellationToken>())
+            .ReturnsAsync(new HttpResponseMessage
+            {
+                StatusCode = HttpStatusCode.OK,
+                Content = new StringContent(JsonSerializer.Serialize(expectedModel))
+            }).Verifiable();
+
+        var httpClient = new HttpClient(_httpMessageHandlerMock.Object);
+        httpClient.BaseAddress = new Uri(BaseAddress);
+
+        var sut = new OrganisationService(httpClient, _logger, _configuration);
+
+        // Act
+        var result = await sut.GetOrganisationRelationshipsByOrganisationId(organisationId);
+
+        // Assert
+        Assert.IsNotNull(result);
+        result.Should().BeEquivalentTo(expectedModel);
+    }
+
+    [TestMethod]
+    public async Task GetOrganisationRelationshipsByOrganisationId_LogsError_AndThrowsException_OnFailure()
```

Comment:
This code doesn't log any error - it just throws an exception

Reply:
Wrote a new test to cover logging